lib/sequelize.js

```diff
@@ -199,6 +199,8 @@ module.exports = (function() {
     this.importCache = {};
 
     Sequelize.runHooks('afterInit', this);
+
+    Promise.prototype.sequelize = this;
```

Comment:
We'd like to support multiple Sequelize instances ideally. We can start without it i suppose but it should be the goal.